website/events-listener/src/index.js

```diff
@@ -0,0 +1,27 @@
+import { GearApi } from '@gear-js/api';
+import { listen } from './events.js';
+import { KafkaProducer } from './producer.js';
+import { logger } from './logger.js';
+import config from './config.js';
+
+const log = logger('Main');
+
+const main = async () => {
+  log.info('App is running...');
+  console.log(config);
```

Comment:
Logging config file is not secure, because it can contain sensitive information

Reply:
Yeah, I forgot to delete this